server/src/database/scripts/sync-chefs-data.ts

```diff
@@ -1,15 +1,20 @@
 import { SyncChefsDataService } from './sync-chefs-data.service';
 import { AppModule } from '../../app.module';
 import { NestFactory } from '@nestjs/core';
+import { SyncTypes } from '../../common/enums';
 
 (async () => {
   const appContext = await NestFactory.createApplicationContext(AppModule);
   const syncChefsDataService = appContext.get(SyncChefsDataService);
   switch (process.env.SYNC_TYPE) {
-    case 'ATTACHMENTS':
+    case SyncTypes.ATTACHMENTS:
       syncChefsDataService.updateAttachments();
       break;
+    case SyncTypes.SUBMISSIONS:
+      syncChefsDataService.syncChefsData(SyncTypes.SUBMISSIONS);
```

Comment:
Call a different method instead of the same! No point in calling the same method, because it'll fetch the form data which will not be needed! We would want a script that updates the submission given the id!